src/test/java/com/google/javascript/clutz/map_entries_with_platform.js

```diff
@@ -0,0 +1,10 @@
+goog.provide('map_entries.Map');
```

Comment:
I try to make all new tests use goog.module as a more realistic to how new JS code looks like. Just a suggestion, not a hard rule.

Reply:
Done.